src/attributes.jl

```diff
@@ -440,6 +440,15 @@ given.
 """
 struct Silent <: AbstractOptimizerAttribute end
 
+"""
+    TimeLimitSec
```

Comment:
`TimeLimitSec()` for consistency with the other docstrings.

Reply:
Should I do the same change to `Silent` too?

Reply:
Yes, thanks.